ckanext/datapusher_plus/jobs.py

```diff
@@ -1374,7 +1386,16 @@ def _push_to_datastore(task_id, input, dry_run=False, temp_dir=None):
         )
         with open(tmp, "rb") as f:
             try:
-                cur.copy_expert(copy_sql, f)
+                # cur.copy_expert(copy_sql, f)
+                subprocess.run(
+                    [
+                        "psql",
+                        "-Atx",
+                        connection_string,
+                        "-c",
+                        f"COPY '{resource_id}' FROM \'" + tmp + "' CSV HEADER",
+                    ],
+                )
```

Comment:
This change doesn't work. 

Even if we fix the syntax error in the query (see bellow), it will get the database in a Lock status and the query will hang.

The former method `cur.copy_expert(...)` works fine. What's the goal of this change?


```python
                subprocess.run(
                    [
                        "psql",
                        "-Atx",
                        connection_string,
                        "-c",
                        f"COPY \"{resource_id}\" FROM \'" + tmp + "' CSV HEADER",
                    ],
                )


```

Reply:
Hey @pdelboca this was part of testing the `COPY` [command](https://wiki.postgresql.org/wiki/COPY) to improve the handling of large files . Using `copy_expert` was really slow comparing with `COPY` when trying to push ~1300000 records into Datastore. 

Local testing were resulting with excellet times, copyng the file in seconds but when this is used into docerized environment we are not getting those performances. So right now this is on hold as we are considering different approaches @jqnatividad

Reply:
That's good to know! Thanks @tino097 !

Looking forward for the version 1.0!